src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -21,6 +21,13 @@
         position: absolute;
         top: $bcs-VersionsItemButton-vertical;
         right: $bcs-VersionsItemButton-horizontal;
+
+        &:not(.is-disabled) {
```

Comment:
small nit but neither `bcs-VersionsItemActions-toggle` or the `PlainButton` uses the `is-disabled` class. that's set for the regular Button component